### ISSUE TITLE
Weekly Office Hours

### ISSUE THREAD

**Issue by alexellis** [openfaas/faas#1683]
Our office hours at 1630 GMT+0 each Wednesday will be open to the community.

https://us02web.zoom.us/j/83845933347?pwd=RzhVeWVHVXJQY2RBM1FMMzd3blhpUT09

Introduce yourself, feel free to bring questions, comments and suggestions.

**Comment by alexellis** [comment]
FYI: there will be no call on 29th Dec 2021.

**Comment by alexellis** [comment]
FYI there's no call on 9th Feb 2022, we will be meeting again 16th Feb. See you there!

**Comment by alexellis** [comment]
We're now meeting at 1630 BST, so GMT+1.

**Comment by bwplotka** [comment]
With @saswatamcode we would like to join tomorrow with the topic: 

* Monitoring requirements for OpenFaaS admin and function users (https://github.com/openfaas/faas/issues/1683)

**Comment by alexellis** [comment]
This week's call is postponed 27th April due to [The Business Of Open Source](https://www.youtube.com/watch?v=wfMJy8FCKF0)

**Comment by alexellis** [comment]
On Wednesday May 18th the call will be postponed due to KubeCon attendance. If you're in Valencia, find me and say hi 👋

**Comment by alexellis** [comment]
@welteki and I are both on leave this week, so the community call will resume next week.

**Comment by alexellis** [comment]
There will be no call on 28th December 2022. Enjoy the holidays :christmas_tree:

**Comment by vorsterk** [comment]
Hi everyone where can i get the information for the openfaas community slack channel?

**Comment by LucasRoesler** [comment]
Hi @vorsterk the Slack channel was discontinued. If you have any community questions, feel free to open issues in Github

**Comment by yuliyantsvetkov** [comment]
Howdy, folks! Are the office hours still ongoing? 

I would love to join to get some insights how I can help, or just hang around with the big brains of OpenFaaS.

**Comment by alexellis** [comment]
Yes, we meet every week. The [link in the docs](https://docs.openfaas.com/community/#weekly-office-hours) lands here.

For questions on commercial use, feel free to reach out to contact@openfaas.com
